app-config.yaml

```diff
@@ -76,23 +76,27 @@ catalog:
       privateToken:
         $secret:
           env: AZURE_PRIVATE_TOKEN
-  exampleEntityLocations:
-    github:
-      - https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/artist-lookup-component.yaml
-      - https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/playback-order-component.yaml
-      - https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/podcast-api-component.yaml
-      - https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/queue-proxy-component.yaml
-      - https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/searcher-component.yaml
-      - https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/playback-lib-component.yaml
-      - https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/www-artist-component.yaml
-      - https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/shuffle-api-component.yaml
-      - https://github.com/spotify/backstage/blob/master/plugins/scaffolder-backend/sample-templates/react-ssr-template/template.yaml
-      - https://github.com/spotify/backstage/blob/master/plugins/scaffolder-backend/sample-templates/springboot-grpc-template/template.yaml
-      - https://github.com/spotify/backstage/blob/master/plugins/scaffolder-backend/sample-templates/create-react-app/template.yaml
-      - https://github.com/spotify/cookiecutter-golang/blob/master/template.yaml
-      - https://github.com/spotify/backstage/blob/master/plugins/scaffolder-backend/sample-templates/docs-template/template.yaml
-      - https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/hello-world-api.yaml
-      - https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/streetlights-api.yaml
+
+  locations:
+    # Backstage example components
+    - type: github
+      target: https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/all-components.yaml
+
+    # Example component for github-actions
+    - type: github
+      target: https://github.com/spotify/backstage/blob/master/plugins/github-actions/examples/sample.yaml
+
+    # Example component for techdocs
+    - type: github
+      target: https://github.com/spotify/backstage/blob/master/plugins/techdocs-backend/examples/documented-component/documented-component.yaml
+
+    # Backstage example APIs
+    - type: github
+      target: https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/all-apis.yaml
+
+    # Backstage example templates
+    - type: github
+      target: https://github.com/spotify/backstage/blob/master/plugins/scaffolder-backend/sample-templates/all-templates.yaml
```

Comment:
Do we need rules for these?

Reply:
Opted for going with an aggregated file, and just have the global rule instead. Rules don't get carried forward in location entities, and it's awkward to implement that atm.